card/mod/05_standard/set/type/search_type.rb

```diff
@@ -231,13 +250,6 @@ def default_search_params
     set_default_search_params :default_limit=>20
   end
   
-  
 end
 
 
```

Comment:
Presumably there is also a standard of generality, so that if there is stuff that should be in the other mod, we split the base functionality from the customization, with super, if we ca.